cabal-testsuite/PackageTests/BuildTools/Foreign/my-foreign-preprocessor.bat

```diff
@@ -0,0 +1,2 @@
+echo @off
+pwsh my-foreign-preprocessor.ps1 %1 %2 %3

```

Comment:
For the reviewer: Perhaps it would be easier to write a BAT sed, but this works :tm: